install/deploy_management_nodes.md

```diff
@@ -194,6 +194,10 @@ proceed to step 2.
 
    1. After the correct time has been verified, power off the NCN.
 
+      ```bash
+      pit# ipmitool -I lanplus -U $USERNAME -E -H $bmc chassis power off
```

Comment:
Aren't the tokens exported at this time? Can this power all the NCNs off instead of just adding the command for a single one?

Reply:
They tokens are already available, but this procedure is written currently to serially power each NCN on one at a time, and have them boot into the BIOS, and then power off the particular NCNs.